crates/uv/tests/common/mod.rs

```diff
@@ -412,7 +413,7 @@ pub fn python_path_with_versions(
                     VersionRequest::from_str(python_version)
                         .expect("The test version request must be valid"),
                 );
-                let sources = SourceSelector::All;
+                let sources = SourceSelector::All(PreviewMode::Enabled);
```

Comment:
We turn on preview so we discover managed toolchains when finding the Python versions requested by tests.